src/power_divergence.jl

```diff
@@ -106,9 +106,10 @@ end
 # adapted from SAS macro by May, Johnson (2000)
 function ci_sison_glaz(x::PowerDivergenceTest, alpha::Float64; skew_correct::Bool=true)
     k = length(x.thetahat)
-    probn = inv(pdf(Poisson(x.n), x.n))
-
+    probn = 1 /(cdf(Poisson(x.n), x.n) - cdf(Poisson(x.n), x.n - 1))
```

Comment:
Why use the `cdf` here instead of the `pdf`?

Reply:
I was trying to figure out where the bug was coming from and so I was making the code as close to the original paper description of the algorithm as I could to narrow down the issue. Turns out the bug was just closer to the bottom of the code. The pdf code should be equivalent here, would you prefer I revert to original line?

Reply:
Yes. Please revert. The previous version is simpler and faster.